Makefile

```diff
@@ -229,3 +230,8 @@ manually-deploy-murm-logging:
 	--set elasticsearch.image=docker.elastic.co/elasticsearch/elasticsearch:8.12.1 \
 	--set kibana.image=docker.elastic.co/kibana/kibana:8.12.1 \
 	--install --atomic --debug
+
+manually-deploy-maintenance:
```

Comment:
I think we don't need DEPLOY_ENV, DOCKER_TAG_PREFIX and SPECIFIC_TAG right?

Reply:
helm upgrade murmurations-maintenance ./charts/murmurations/charts/maintenance  --install --atomic --debug

Reply:
We may not require DEPLOY_ENV, but DOCKER_TAG_PREFIX and SPECIFIC_TAG are essential, as the image name varies. For instance, in the pretest environment, the image name changes to prefix-maintenance. Regarding SPECIFIC_TAG, it's necessary because we need it in case we need a specific tag like other services.

Reply:
OK